packages/template-ui/src/views/Search.vue

```diff
@@ -146,6 +145,28 @@ export default {
       return this.onNodeUpdated(nodeId, this.resultNodes);
     },
   },
+  $trs: {
+    onlyDownloadedItems: {
+      message: 'Only downloaded items',
+      context: 'Label for a search filter checkbox',
+    },
+    noContentFound: {
+      message: 'Sorry, we can’t find any content that matches your search.',
+      context: 'Text shown to the user when a search returns no results',
+    },
+    noContentFoundGuidance: {
+      message: 'You can try a different search, maybe use fewer words, or try one of the topic suggestions below.',
+      context: 'Guidance given to the user when no search results are found',
+    },
+    exploreTopics: {
+      message: 'Explore topics',
+      context: 'Heading for a section of content grid in the search results',
+    },
+    resultsCount: {
+      message: '{num_results} Results',
```

Comment:
```suggestion
      message: '{num_results, number, integer} {num_results, plural, one {Result} other {Results}}',
```

We can be a bit fancier now this is going through the translation stuff :)